ledger/src/blockstore.rs

```diff
@@ -10692,44 +10692,52 @@ pub mod tests {
     fn erasure_multiple_config() {
         solana_logger::setup();
         let slot = 1;
-        let parent = 0;
         let num_txs = 20;
-        let entry = make_large_tx_entry(num_txs);
-        let shreds = entries_to_test_shreds(
-            &[entry],
-            slot,
-            parent,
-            true,  // is_full_slot
-            0,     // version
-            false, // merkle_variant
-        );
-        assert!(shreds.len() > 1);
+        let entries = [make_large_tx_entry(num_txs)];
 
-        let ledger_path = get_tmp_ledger_path_auto_delete!();
-        let blockstore = Blockstore::open(ledger_path.path()).unwrap();
+        let version = solana_sdk::shred_version::version_from_hash(&entries[0].hash);
```

Comment:
convention is usually to import `version_from_hash` or `shred_version`.

Reply:
Copied that line from another spot in this same file, but yeah, new code needs not be as ugly as old one!

Reply:
yeah, this is still an incremental step in the right direction, so I'm fine with it as is

Reply:
I've added the import to the updated code. Do not think churning the old code now is justified.